RoslynAnalyzers.sln

```diff
@@ -166,7 +166,6 @@ Global
 		src\Utilities\Workspaces\Workspaces.Utilities.projitems*{1c62a2f6-7a44-4030-9d1a-7e7a966736e5}*SharedItemsImports = 5
 		src\Utilities\Compiler\Analyzer.Utilities.projitems*{2a4af3a4-307b-4252-888e-e6546244585a}*SharedItemsImports = 5
 		src\Utilities\Workspaces\Workspaces.Utilities.projitems*{2a4af3a4-307b-4252-888e-e6546244585a}*SharedItemsImports = 5
-		src\Utilities\Compiler\Analyzer.Utilities.projitems*{2a520f3e-8c5d-43fe-aa03-fe5e3c5f23d1}*SharedItemsImports = 5
```

Comment:
@Evangelink Even I am seeing these now :) Probably best to just update the sln?

Reply:
I am not alone \o/ ! I don't know about the update... I have seen that we end-up with less projects referenced than in some other/older branches so I am wondering if something is broken.